elasticsearch_test.go

```diff
@@ -47,105 +45,78 @@ func (e *UnitTestEnv) TestElasticsearch_Type(t *testing.T) {
 	}
 }
 
-func (e *UnitTestEnv) TestElasticsearch_Init(t *testing.T) {
-	config := map[string]interface{}{
-		"username": e.Username,
-		"password": e.Password,
-		"url":      e.URL,
-	}
-	configToStore, err := e.Elasticsearch.Init(context.Background(), config, true)
-	if err != nil {
-		t.Fatal(err)
+func (e *UnitTestEnv) TestElasticsearch_Initialize(t *testing.T) {
+	req := newdbplugin.InitializeRequest{
+		Config: map[string]interface{}{
+			"username": e.Username,
+			"password": e.Password,
+			"url":      e.URL,
+		},
+		VerifyConnection: true,
 	}
-	if len(configToStore) != len(config) {
-		t.Fatalf("expected %s, received %s", config, configToStore)
+	resp := dbtesting.AssertInitialize(t, e.Elasticsearch, req)
+	if len(resp.Config) != len(req.Config) {
+		t.Fatalf("expected %s, received %s", req.Config, resp.Config)
 	}
-	for k, v := range config {
-		if configToStore[k] != v {
-			t.Fatalf("for %s, expected %s but received %s", k, v, configToStore[k])
+	for k, v := range req.Config {
+		if resp.Config[k] != v {
+			t.Fatalf("for %s, expected %s but received %s", k, v, resp.Config[k])
 		}
 	}
```

Comment:
Can this be replaced with `if !reflect.DeepEqual(resp.Config, req.Config)` ?
Same question in some of the other tests I see this in

Reply:
Yep, seems to work, changed in 507cc5319bdb4adbd706ad866cd591d087de9dfc